code/game/objects/items/weapons/his_grace.dm

```diff
@@ -20,14 +20,14 @@
 	var/prev_bloodthirst = HIS_GRACE_SATIATED
 	var/force_bonus = 0
 
-/obj/item/weapon/his_grace/New()
+/obj/item/weapon/his_grace/Initialize()
 	..()
 	START_PROCESSING(SSprocessing, src)
 
 /obj/item/weapon/his_grace/Destroy()
 	STOP_PROCESSING(SSprocessing, src)
 	for(var/mob/living/L in src)
-		L.forceMove(get_turf(src))
+		L.forceMove(loc)
```

Comment:
This will end up with bodies inside of, like, the research deconstructor. That would be bad. You're supposed to be able to get the bodies back without too much hassle just by destroying it and I don't actually know what would happen if you suddenly filled the research deconstructor with bodies.